core/src/main/java/org/apache/accumulo/core/client/ConnectionInfo.java

```diff
@@ -35,7 +36,7 @@
   /**
    * @return Zookeeper connection information for Accumulo instance
    */
-  String getZookeepers();
+  String getZooKeepers();
```

Comment:
Capital K?

Reply:
That's how we have referenced them in the past and how Apache ZooKeeper spells it. I can switch back but @ctubbsii might disagree :-)

Reply:
If it's public API, we should follow upstream naming conventions. Internally, it doesn't really matter, other than as best practice for developing the habit of using the trademark properly.

Reply:
Probably best for another PR